src/components/screens/home/balance-card.tsx

```diff
@@ -0,0 +1,72 @@
+import { getHoursAndMinutes } from "../../../utils/time-utils";
+import { Grid, Typography, Card, CardContent, Skeleton } from "@mui/material";
+import strings from "../../../localization/strings";
+import ScheduleIcon from "@mui/icons-material/Schedule";
+import { errorAtom } from "../../../atoms/error";
+import { useAtomValue, useSetAtom } from "jotai";
+import { useEffect, useState } from "react";
+import { useApi } from "../../../hooks/use-api";
+import { authAtom, userProfileAtom } from "../../../atoms/auth";
+
+/**
+ * Component for displaying user's balance
+ */
+const BalanceCard = () => {
+  const userProfile = useAtomValue(userProfileAtom);
+  const { personsApi } = useApi();
+  const setError = useSetAtom(errorAtom);
+  const [isLoading, setIsLoading] = useState(true);
+  const [result, setResult] = useState("");
+
+  /**
+   * Initialize logged in person's time data.
+   */
+  const getPersons = async () => {
+    const fetchedPersons = await personsApi.listPersons({ active: true });
```

Comment:
isLoading should be set as true at the start of the request, so in this case, the first line of the getPersons funciton